Minor polish and update of doc/quickstart/r.md (#4228)
mlpack/mlpack: doc/quickstart/r.md

Building the R bindings from scratch is a little more in-depth, though.  For
information on that, follow the instructions in the
[installation guide](../user/install.md#compile-bindings-manually). Note that

> the mlpack R binding turn off support for the STB, DR_LIBS, and HTTPLIB

Reviewer: ```suggestion
by default, mlpack's R bindings disable support for the STB, dr_libs, and httplib
```

> libraries which may be of lesser interst to R users. If desired, support at

Reviewer: ```suggestion
libraries, as they are not used in any of the R bindings. If desired, support at
```

Author: This, as well as the other suggestions that follow, are all good.  I just applied them.

> the C++ level can be enabled by setting per-library `#define` statements.

Reviewer: ```suggestion
the C++ level can be enabled by setting per-library `#define` statements; see
`inst/include/mlpack.h` in the generated package.
```

What do you think, maybe good to give a breadcrumb here and point them in the right direction?

## Simple mlpack quickstart example

As a really simple example of how to use mlpack from R, let us do some
simple classification on a subset of the standard machine learning `covertype`
dataset.  We will first split the dataset into a training set and a testing set,
then we will train an mlpack random forest on the training data, and finally

> we will print the accuracy of the random forest on the test dataset.

Reviewer: I don't mind the wording changes here at all, but, do you think you could also deploy them to the other quickstarts so we can keep the language generally in sync?  I think this paragraph shows up word-for-word in the others.

Author: Do you mean my minor language policing of "we'll" into "we will"?  Could do.

Author: Done for the other files doc/quickstart/*.md and their use of "we'll".
